docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc

```diff
@@ -1398,26 +1455,38 @@ a9:a3:21:30:1f:30:1d:06:03:55:1d:0e:04:16:04:14:5e:0d:77:0b:55:63:c1:20:8b:a1:a6
 
 [source,options="nowrap"]
 ----
-/subsystem=elytron/key-store=httpsKS:read-alias(name="localhost",verbose=false)
+/subsystem=elytron/key-store=httpsKS:read-alias(alias="localhost",verbose=false)
 {
     "outcome" => "success",
     "result" => {
-            "alias" => "localhost",
```

Comment:
@theashiot  I see, previously the documentation was incorrect since you can see in the above command that the RSA is being specified,`algorithm=RSA` . Thanks for fixing it!

Reply:
oh right, i didn't notice the command :)